qa/tasks/cephadm.py

```diff
@@ -100,6 +100,43 @@ def update_archive_setting(ctx, key, value):
         yaml.safe_dump(info_yaml, info_file, default_flow_style=False)
 
 
+@contextlib.contextmanager
+def nvmeof_gateway_cfg(ctx, config):
+    source_host = config.get('source')
+    target_host = config.get('target')
+    nvmeof_service = config.get('service')
+    assert source_host
+    assert target_host
+    assert nvmeof_service
```

Comment:
Bare asserts don't lend themselves to useful error messages; I'd suggest either adding a message to these asserts, or doing a little more verification and e.g. raising a `ValueError` with an informative message.

Reply:
Makes sense, thank you! I raised `ConfigError` exception instead of bare asserts here.